PMAssist/Models/EventSourceApi.cs

```diff
@@ -5,5 +5,7 @@ public class EventSourceApi
         public string? id { get; set; }
         public string? url { get; set; }
         public string? format { get; set; }
+        public string? startStr { get; set; }
```

Comment:
@GajananTuppad can the dates be Date Types

Reply:
These variable to show date string in edit screen without default time stamp.